### PR TITLE
fix(theme): revert sass:math.div

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

we have tried to switch to @use sass directive but we fails with too many issues.
We will run that later as breaking change.

Also I found that theme do not start any more:
```
jmfrancois@LT-C02WN2D6HTDF ~/github/talend/ui/packages/theme (master)$ yarn start
yarn run v1.22.11
$ webpack serve --mode=development
[webpack-cli] Invalid options object. Dev Server has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'inline'. These properties are valid:
   object { allowedHosts?, bonjour?, client?, compress?, devMiddleware?, headers?, historyApiFallback?, host?, hot?, http2?, https?, ipc?, liveReload?, magicHtml?, onAfterSetupMiddleware?, onBeforeSetupMiddleware?, onListening?, open?, port?, proxy?, setupExitSignals?, static?, watchFiles?, webSocketServer? }
error Command failed with exit code 2.
```

**What is the chosen solution to this problem?**

Revert and use css calc to avoid the log

**Please check if the PR fulfills these requirements**

* [x] The PR have used `yarn changeset` to a request a release from the CI if wanted.
* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: workspace-run.js [hunk 1]
```diff
@@ -30,13 +30,19 @@ function run(cmd, opts = {}) {
...
 			if (opts.verbose && stdout) {
-				console.error(`RUNNER: Child Process STDOUT: ${stdout}`);
+				console.error(`#### RUNNER: Child Process STDOUT: ${stdout}`);
 			}
+			if (code > 0) {
+				console.error(`#### RUNNER: ${cmd.name} ${cmd.args.join(' ')} exit code ${code}`);
+				reject(stderr);
+				return;
+			}
+			console.log(`#### RUNNER: ${cmd.name} ${cmd.args.join(' ')} exit code ${code}`);
```
Comment (jmfrancois) [inline workspace-run.js:45]: bonus we fix the exit code of workspace-run. If during pre-release one script fails it continues 😮‍💨

**Review by jsomsanith-tlnd — APPROVED** [review]

File: packages/components/src/Badge/Badge.scss [hunk 2]
```diff
@@ -1,10 +1,8 @@
-@use 'sass:math';
-
 $tc-badge-large-label-font-size: 1.4rem !default;
 $tc-badge-large-label-font-weight: normal !default;
 $tc-badge-large-label-with-categ-font-weight: $font-weight-semi-bold !default;
 $tc-badge-large-height: 2.4rem !default;
-$tc-badge-large-vertical-padding: math.div($tc-badge-large-height - $tc-badge-large-label-font-size * $line-height-base, 2) !default;
+$tc-badge-large-vertical-padding: $tc-badge-large-height - $tc-badge-large-label-font-size * 0.7142857145 !default; // $line-height-base / 2
```
Comment (romainseb) [inline packages/components/src/Badge/Badge.scss:5]: Outch 😬

File: packages/theme/webpack.config.js [hunk 3]
```diff
@@ -87,11 +87,13 @@ module.exports = (env, argv) => {
 		],
 		devServer: {
 			port: 1234,
-			stats: 'errors-only',
+			// stats: 'errors-only',
```
Comment (romainseb) [inline packages/theme/webpack.config.js:90]: Maybe it can be removed instead of be commented ?

File: packages/theme/webpack.config.js [hunk 4]
```diff
@@ -87,11 +87,13 @@ module.exports = (env, argv) => {
...
 			historyApiFallback: true,
-			contentBase: path.join(process.cwd(), 'dist'),
+			static: {
+				directory: path.join(process.cwd(), 'dist'),
+			},
+			// contentBase: path.join(process.cwd(), 'dist'),
```
Comment (romainseb) [inline packages/theme/webpack.config.js:95]: Maybe it can be removed instead of be commented ?

**Review by sgangnant — APPROVED** [review]

**Review by romainseb — APPROVED** [review]

File: packages/theme/webpack.config.js (hunk 3 above)
Comment (Kporal) [inline packages/theme/webpack.config.js:90]: Why this change?

File: packages/theme/webpack.config.js (hunk 4 above)
Comment (Kporal) [inline packages/theme/webpack.config.js:95]: Same here?

Comment (jmfrancois) [reply packages/theme/webpack.config.js:95]: added in PR desciption

File: packages/components/src/Badge/Badge.scss (hunk 2 above)
Comment (jmfrancois) [reply packages/components/src/Badge/Badge.scss:5]: good catch

Comment (jmfrancois) [reply packages/components/src/Badge/Badge.scss]: fixed thans !
